frontend/smart-kickers-game/src/apis/Game.spec.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { resetGame } from './Game';
+
+describe('resetGame API caller', () => {
+  let mock;
+
+  beforeAll(() => {
+    mock = new MockAdapter(axios);
+  });
+
+  afterEach(() => {
+    mock.reset();
+  });
+
+  it('should send reset game request', async () => {
+    mock.onPost(`http://localhost:3000/reset`).reply(200);
```

Comment:
I'd suggest to introduce a global configuration variable to store the backend URL origin. Now if you change e.g. the port, all your tests and `fetch`s will need to be adjusted.